webapp-django/crashstats/cron/management/commands/cronrun.py

```diff
@@ -122,56 +123,64 @@ def _run_one(self, job_spec, force=False, cmd_args=None):
         start_time = None
         run_time = None
 
-        try:
-            cmd_kwargs = {}
-            with self.lock_job(job_spec['cmd']):
+        with self.lock_job(job_spec['cmd']):
+            try:
+                cmd_kwargs = {}
+                last_success = job.last_success
+
                 # Backfill jobs can have multiple run-times, so we iterate
                 # through all possible ones until either we get them all done
                 # or it dies
                 for run_time in get_run_times(job_spec, job.last_success):
-                    # If "backfill" is in the spec, then we want to additionally
-                    # pass in a run_time argument
+                    if job_spec.get('backfill', False):
+                        # If "backfill" is in the spec, then we want to pass in
+                        # run_time as an argument
+                        cmd_kwargs['run_time'] = format_datetime(run_time)
```

Comment:
These arguments get passed in as strings so the command code can parse them just like any argument.